src/member/signals/handlers.py

```diff
@@ -0,0 +1,10 @@
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+
+from ..models import ClientScheduledStatus
+
+
+@receiver(post_delete, sender=ClientScheduledStatus)
+def ensure_pair_remove(sender, instance, **kwargs):
+    if instance.get_pair:
+        ClientScheduledStatus.objects.get(pk=instance.get_pair.pk).delete()
```

Comment:
Discussion: if a pair status change has been performed, for example, yesterday, do we delete it anyways or keep it in the database?

A related question to discuss: do we allow the user to delete a status change that has happened?